### DISCUSSION TITLE
Refactor hyperspec.lisp to be generic over CLCS and NovaSpec

### DISCUSSION THREAD

**Discussion by jgarte** [lem-project/lem#1544] [Ideas]
Hi,

Would a refactor be accepted to make [hyperspec.lisp](https://github.com/lem-project/lem/blob/main/extensions/lisp-mode/ext/hyperspec.lisp) generic over [CLCS](https://cl-community-spec.github.io/pages/index.html) and [NovaSpec](https://novaspec.org/cl/)?

**Comment by vindarel** [comment]
hello, what do you have in mind and do you have a POC?

Is pointing the `*hyperspec-root*` to CLCS enough, or does that need more work?

**Reply by jgarte** [reply]
That's a good question. 

I think it may not be enough as the paths are different, if I remember correctly.

I'll have to do some more research on your question. I'll look into it and we can discuss more about the topic here once I or someone else finds out.
